src/main/java/uk/gov/hmcts/reform/professionalapi/repository/OrganisationRepository.java

```diff
@@ -3,8 +3,10 @@
 import java.util.List;
 import java.util.UUID;
 
+import feign.Param;
```

Comment:
@ishratt Wrong import is used for the `@param` annotation. Please correct this to - `org.springframework.data.repository.query.Param`

Reply:
thanks for spotting this, will amend it.